program/convert-minutes-to-weeks/minutes-to-week.py

```diff
@@ -0,0 +1,6 @@
+
+if __name__ == '__main__':
```

Comment:
What is the purpose of this line? It is blocking the code from running.

Reply:
It is Python main function and working fine

```bash
10080
1
```